### ISSUE TITLE
Spurious log message about failing to clear cache for realms on startup

### ISSUE THREAD

**Issue by dakrone** [elastic/elasticsearch#35218]
When Elasticsearch starts up, the following error message is usually printed (without any sort of security configuration):

```
[2018-11-02T11:23:36,061][INFO ][o.e.n.Node               ] [hot1] started                                                                      
[2018-11-02T11:23:39,391][WARN ][o.e.x.s.a.s.m.NativeRoleMappingStore] [hot1] Failed to clear cache for realms [[]]                       
```

It would be great if we could track down the failure and remove the error message (I'm assuming it's probably a byproduct of the startup sequence or the index not being ready or something)

**Comment by amrapalijz** [comment]
Has this problem been resolved?

**Comment by dayuebai** [comment]
Still seeing this warning when I use Elasticsearch 7.0. Should I worry about it or just ignore this warning message?

**Comment by tvernum** [comment]
If it is that exact message with `[[]]` at the end, then you can safely ignore it.

**Comment by SiwyDym** [comment]
@tvernum  I have `Failed to clear cache for realms [[]]` as you wrote I can ignore that, but why they are logged ?

**Comment by Chealer** [comment]
I am experiencing this with ElasticSearch 6.8.13. My understanding is that this is only resolved in ES 7.2 and ulterior.
